src/services/groups.ts

```diff
@@ -39,3 +40,47 @@ export function generateSecret(wordlist: string[], length: number): string {
   }
   return mnemonicWords.join('-');
 }
+
+/**
+ * Executes a query to retrieve the members of a group.
+
+ * @param {PostgresJsDatabase<typeof db>} dbPool - The database connection pool.
+ * @param {string} groupId - The ID of the user.
+ */
+export async function getGroupMembers(dbPool: PostgresJsDatabase<typeof db>, groupId: string) {
+  const response = await dbPool.query.groups.findMany({
+    where: eq(db.groups.id, groupId),
+    with: {
+      usersToGroups: {
+        with: {
+          user: true,
+        },
+      },
+    },
+  });
+
+  // Extract user objects from each usersToGroups entry
+  const groupMembers = response.flatMap((group) =>
+    group.usersToGroups.map((usersToGroup) => {
+      const { telegram, createdAt, updatedAt, username, email, ...userWithoutSensitiveInfo } =
+        usersToGroup.user;
+      return userWithoutSensitiveInfo;
```

Comment:
lets leave username in the userWithoutSensitiveInfo object

Reply:
Ok